### ISSUE TITLE
Labels are not visible in combination with plugin "UI Customizer"

### ISSUE THREAD

**Issue by fank** [catgiggle/OctoPrint-GpioControl#22]
The background as well as the icon is "white":   
![image](https://user-images.githubusercontent.com/1900106/110517232-e2396380-810a-11eb-9ab3-929d546ff6b8.png)

Like seen in the "Top Temp" plugin the theme should looks like:
![image](https://user-images.githubusercontent.com/1900106/110517596-583dca80-810b-11eb-8385-3c93f38d6889.png)

**Comment by catgiggle** [comment]
Good point! I'm also using "UI Customizer" (really good plugin!) but not for black theme, therefore I never noticed this problem. I wil try to fix this with next version.
